### PR TITLE
Adds automatic dir-facing on fire + a few balloon alerts + spellcheck

### PR DESCRIPTION
**Pull request by aidan-bowman**
## About the Pull Request

Adds an OPTIONAL pref for automatically facing direction after firing a gun.

Also converts a few gun-related to_chats to balloon_alerts

Also also fixes a pref's spelling

## Why It's Good For The Game

The auto-facedir was suggested as a nice QoL feature, to make it easier to shoot while retreating.

## Changelog

:cl:
qol: New preference: "Auto-face direction after shooting". Now you can retreat while firing WITHOUT having to struggle with Bay controls!
qol: A few gun-related to_chats (on the sidebar) have been changed to balloon_alerts (right on top of the gun).
balance: rebalanced something
spellcheck: Fixes submit/cancel swap button typo.
/:cl:

### CODE REVIEW

**Comment by 77seven77seven** [comment]
closes #1303

**Comment by aidan-bowman** [comment]
> closes #1303 

No it doesn't

File: code/modules/projectiles/gun.dm [hunk 1]
```diff
@@ -270,8 +274,7 @@
 			return
 
 	if(world.time < next_fire_time)
-		if (world.time % 3) //to prevent spam
-			to_chat(user, SPAN_WARNING("[src] is not ready to fire again!"))
+		balloon_alert(user, "can't fire!")
```
Comment (TheDarkElites) [inline code/modules/projectiles/gun.dm]:
```suggestion
		balloon_alert(user, "can't fire yet!")
```

**Comment by ghost** [comment]
There's an idea but Uh, could you tie it so that whlist on harm intent the char face is going to track the mouse dir rather than be specifically after a ranged weapon use?

**Comment by Tsurupeta** [comment]
Just add the interface button to face dir

**Review by TheDarkElites — CHANGES_REQUESTED** [review]

**Comment by aidan-bowman** [comment]
> Just add the interface button to face dir

out of scope

**Comment by Tsurupeta** [comment]
> > Just add the interface button to face dir
> 
> out of scope

I don't like this PR, you're approaching problem from the wrong direction.

**Comment by aidan-bowman** [comment]
> > > Just add the interface button to face dir
> > 
> > 
> > out of scope
> 
> I don't like this PR, you're approaching problem from the wrong direction.

???? this isn't stopping us from adding a button for facing dir as well, it's just a clientside pref that some people might find useful.

**Review by TheDarkElites — APPROVED** [review]

**Comment by TheDarkElites** [comment]
Despite Tsurpta's protests I think this PR is fine and the preference is optional so I dont see an issue.

**Pull request merged**